v1-1-RC1/CipherList.md

```diff
@@ -0,0 +1,52 @@
+# FIXS Recomended Cipher List
+## Purpose
+This list is the reference list of approved ciphers for use with FIXS
+
+Ciphers are listed in groups under sub-headings, first by TLS Version and then by use-case
+
+- [FIXS Recomended Cipher List](#fixs-recomended-cipher-list)
+    - [Purpose](#purpose)
+    - [Overview](#overview)
+    - [Revision notes](#revision-notes)
+    - [TLS1.2](#tls12)
+        - [Authentication](#authentication)
+            - [TLS Certificate Authentication](#tls-certificate-authentication)
+            - [TLS PSK Authentication](#tls-psk-authentication)
+
+## Overview
+The acronyms used in this appendix match those in the TLS Cipher Suite Registry which is authoritative and held at IANA for IETF. We have chosen to use these acronyms as opposed to, for example, using the ones from the OpenSSL cipher suite names. See
+http://www.iana.org/assignments/tls-parameters/tls-parameters.xhtml or https://www.ietf.org/assignments/tls-parameters/tls-parameters.txt
+
+## Revision notes
+For a full list of revisions refer to the git commit history
+| Version | Note
+|---------|------
+| Initial | First commit, replicates FIXS V1.0 text
+| 2021.1  | Updated to remove CBC and SHA1 as these deprecated for use with TLS1.2 and above.
+
+## TLS1.2
+
+### Authentication
+#### TLS Certificate Authentication
+The following cipher suite list when using certificates for authentication. This includes using certificates in Simple TLS in conjunction with FIXA. The list ensures Forward Secrecy, avoids deprecated ciphers and should achieve good performance. The cipher suites are specified in our order of preference, starting with the most preferred cipher suite.
```

Comment:
> The following cipher suite list when using certificates for authentication.

I am sure there is something missing between "list" and "when". Maybe "should be used" or "should be considered"?